rotkehlchen/chain/ethereum/modules/ens/decoder.py

```diff
@@ -170,52 +207,63 @@ def _decode_ens_registry_with_fallback_event(self, context: DecoderContext) -> D
             ))
         return DEFAULT_DECODING_OUTPUT
 
-    def _decode_ens_public_resolver_2_events(self, context: DecoderContext) -> DecodingOutput:
+    def _decode_ens_public_resolver_events(self, context: DecoderContext) -> DecodingOutput:
         """Decode event where a text property (discord, telegram, etc.) is set for an ENS name."""
-        if context.tx_log.topics[0] == TEXT_CHANGED:
-            try:
-                _, decoded_data = decode_event_data_abi_str(context.tx_log, TEXT_CHANGED_ABI)
-                changed_key = decoded_data[0]
-            except (DeserializationError, KeyError) as e:
-                msg = str(e)
-                if isinstance(e, KeyError):
-                    msg = f'missing key {msg}'
-
-                log.debug(f'Failed to decode ENS set-text event due to {msg}')
-                return DEFAULT_DECODING_OUTPUT
+        if context.tx_log.topics[0] not in (TEXT_CHANGED_1, TEXT_CHANGED_2):
+            return DEFAULT_DECODING_OUTPUT
 
-            node = context.tx_log.topics[1]
-            try:
-                address = self.ethereum.contracts.contract('ENS_PUBLIC_RESOLVER_2').call(
-                    node_inquirer=self.ethereum,
-                    method_name='addr',
-                    arguments=[node],
-                )
-                address = to_checksum_address(address)
-                ens_mapping = find_ens_mappings(
-                    ethereum_inquirer=self.ethereum,
-                    addresses=[address],
-                    ignore_cache=False,
-                )
-            except RemoteError as e:
-                log.debug(f'Failed to decode ENS set-text event due to {str(e)}')
-                return DEFAULT_DECODING_OUTPUT
+        try:
+            _, decoded_data = decode_event_data_abi_str(
+                context.tx_log,
+                TEXT_CHANGED_1_ABI if context.tx_log.topics[0] == TEXT_CHANGED_1 else TEXT_CHANGED_2_ABI,  # noqa: E501
+            )
+            changed_key = decoded_data[0]
+            new_value = decoded_data[1] if context.tx_log.topics[0] == TEXT_CHANGED_2 else None  # noqa: E501
+        except (DeserializationError, KeyError) as e:
+            msg = str(e)
+            if isinstance(e, KeyError):
+                msg = f'missing key {msg}'
```

Comment:
Hmm yeah, I also don't know, it was like that before. Maybe @LefterisJP knows.

Handling index error I believe is not needed since the event is either decoded according to its abi and has all the indexes or a deserialization error is raise.